requirements.txt

```diff
@@ -11,8 +11,8 @@ git+https://github.com/dwf/picklable_itertools.git#egg=picklable-itertools
 
 --allow-external theano
 --allow-unverified theano
-git+https://github.com/Theano/Theano.git#egg=theano
+git+https://github.com/Theano/Theano.git@c6d2ab3929182cdf7cf12e4b1f191a6ff99d07cc#egg=theano
```

Comment:
I even know what the first "hotfix" in terminology from http://nvie.com/posts/a-successful-git-branching-model/ will be: changing Theano version to the one with https://github.com/Theano/Theano/issues/2913 fixed.